releasing/README.md

```diff
@@ -135,6 +136,10 @@ this should be automated, and descriptions in PR's should
 be standardized to make automation possible.
 See kubebuilder project.
 
+#### Fetching New OpenAPI Data
```

Comment:
Please make this "Consider fetching ..."
and move it up to be the first step in the Release Sequence, 
right before "Establish clean slate".